scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -297,6 +294,35 @@ task Add_AS_MAX_VQS_FIELD_ToVcf {
   }
 }
 
+task IsVQSRLite {
```

Comment:
I feel like "CheckIfVQSRLite" would be more helpful, but I'm not against the current name

Reply:
Yeah.